vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WaitSteps.java

```diff
@@ -394,6 +397,27 @@ public void waitForScroll()
         javascriptActions.waitUntilScrollFinished();
     }
 
+    /**
+     * Sets a custom timeout for loading all following pages
+     * <br>
+     * Usage example:
+     * <code>
+     * <br>Given I am on a page with the URL '${vividus-test-site-url}/elementState.html'
+     * <br>When I set page load timeout to `PT15S`
+     * <br>When I open URL `${vividus-test-site-url}/delayedLoading?imageTimeout=10000` in new window
+     * <br>When I set page load timeout to `PT10S`
+     * </code>
+     * <br>
+     * @param duration timeout for pages loading
+     */
+    @When("I set page load timeout to `$duration`")
+    public void configurePageLoadTimeout(Duration duration)
+    {
+        timeoutConfigurer.setPageLoadTimeout((int) duration.getSeconds());
+        timeoutConfigurer.setPageLoadTimeoutTimeUnit(TimeUnit.SECONDS);
```

Comment:
Keep bean state unchanged: just create a new method that allows to set page load timeout using the provided parameters: `timeoutConfigurer.configurePageLoadTimeout(Duration)`